### PR TITLE
updating to run tests in ci space

### PR DESCRIPTION
**Pull request by rhiGriff**
This PR is only to get the tests running in the ci space. If you are to only run the tests you now only need to run make acceptance_tests as this also loads the data for the ci space

### CODE REVIEW

File: acceptance_tests/features/steps/send_message_from_todo_list.py.orig [hunk 1]
```diff
@@ -0,0 +1,118 @@
+from behave import given, when, then
```
Comment (gemmai95) [inline acceptance_tests/features/steps/send_message_from_todo_list.py.orig]: Try taking this out. I don't think you need it

File: resources/database/database_reset_rm.sql.orig [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+TRUNCATE collectionexercise.sampleunit CASCADE;
+TRUNCATE collectionexercise.sampleunitgroup CASCADE;
+TRUNCATE collectionexercise.samplelink CASCADE;
+TRUNCATE collectionexercise.collectionexercise CASCADE;
+
+ALTER SEQUENCE collectionexercise.sampleunitgrouppkseq RESTART WITH 1;
+ALTER SEQUENCE collectionexercise.sampleunitpkseq RESTART WITH 1;
+ALTER SEQUENCE collectionexercise.samplelinkpkseq RESTART WITH 1;
+
+<<<<<<< HEAD
```
Comment (gemmai95) [inline resources/database/database_reset_rm.sql.orig]: Seems to be a merge conflict not dealt with here

File: data/clean-database-2018-03-08.sql [hunk 3]
```diff
@@ -5575,7 +5575,7 @@ COPY casetypeoverride (casetypeoverridepk, exercisefk, sampleunittypefk, actionp
 --
 
 COPY collectionexercise (id, exercisepk, name, scheduledstartdatetime, scheduledexecutiondatetime, scheduledreturndatetime, scheduledenddatetime, periodstartdatetime, periodenddatetime, actualexecutiondatetime, actualpublishdatetime, executedby, statefk, samplesize, exerciseref, user_description, created, updated, deleted, survey_uuid) FROM stdin;
-14fb3e68-4dca-46db-bf49-04b84e07e77c	1	BRES_2017	2017-09-11 23:00:00+00	2017-09-10 23:00:00+00	2017-10-06 00:00:00+00	2018-06-29 23:00:00+00	2017-09-14 23:00:00+00	2017-09-15 22:59:59+00	\N	\N	\N	CREATED	\N	221_201712	\N	\N	\N	f	cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87
+§fb3e68-4dca-46db-bf49-04b84e07e77c	1	BRES_2017	2017-09-11 23:00:00+00	2017-09-10 23:00:00+00	2017-10-06 00:00:00+00	2018-06-29 23:00:00+00	2017-09-14 23:00:00+00	2017-09-15 22:59:59+00	\N	\N	\N	CREATED	\N	221_201712	\N	\N	\N	f	cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87
```
Comment (tej99) [inline data/clean-database-2018-03-08.sql]: §? typo?

File: Makefile [hunk 4]
```diff
@@ -20,11 +20,14 @@ stop_services:
...
+	pipenv run python reset_database.py
+
+setup: clean
 	./setup_data.sh ${RM_TOOLS_REPO_URL}
 
 system_tests:
 	pipenv run behave --format progress2 system_tests/features # This will only run the system tests
 
-acceptance_tests:
+acceptance_tests: setup
```
Comment (tej99) [inline Makefile]: Not sure if setup should be ran every time acceptance_tests is. Set up is only needed once at the start to load CEs and events. Maybe acceptance_tests: clean?

**Pull request merged**